lgtm.yml

```diff
@@ -1,3 +1,10 @@
+path_classifiers:
+  test:
+    - exclude: tests
+    - exclude: tools
+  benchmark:
+    - exclude: benchmarks
```

Comment:
I see `test` at https://lgtm.com/help/lgtm/file-classification#built-in-tags but not `benchmark`.  Where is `benchmark` documented?

Reply:
I just see the tag in the file browser: https://lgtm.com/projects/g/Chia-Network/chia-blockchain/latest/files/benchmarks/?sort=name&dir=ASC&mode=heatmap